e2e/config.json

```diff
@@ -0,0 +1,4 @@
+{
+	"setupTestFrameworkScriptFile": "./init.js",
```

Comment:
Do we want tabs? Or two-space as is the custom with `package.json`?

Reply:
Tabs for consistency with everything that's not package.json

Reply:
(Plus, tab is 1 char, 2 spaces is 2 chars. 1 < 2, so 1 is better :p)

Reply:
Alright, I will not argue with something that I agree with. Tabs are better.